autogpt_platform/frontend/src/components/integrations/credentials-input.tsx

```diff
@@ -84,6 +84,7 @@ export const providerIcons: Record<
   reddit: fallbackIcon,
   fal: fallbackIcon,
   revid: fallbackIcon,
+  proxycurl: fallbackIcon,
```

Comment:
alphabetically this isn't in the right place.. although I see it's not the only one